src/domain/SuggestionPage/fields/TranslatedFieldInput.tsx

```diff
@@ -54,10 +52,14 @@ const TranslatedFieldInput: React.FC<TranslatedFieldInputProps> = ({
             value={value.sv}
             onChange={handleChange}
             invalid={!!error}
-            helperText={error}
           />
         </Tab>
       </Tabs>
+      {error && (
+        <span className={styles.formError}>
+          {error.fi ?? error.en ?? error.sv}
```

Comment:
```suggestion
          {Object.keys[error].find(msg => !!msg)}
```
Maybe? 🤔

Reply:
I'd prefer to keep them in the same order the inputs are.